app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,18 @@
+module SessionsHelper
+  def log_in user
+    session[:user_id] = user.id
+  end
+
+  def current_user
+    @current_user ||= User.find_by id: session[:user_id]
```

Comment:
toán tử `||=` có tác dụng gì?

Reply:
Toán tử `| |= `có tác dụng kiểm tra xem biến `@current_user` đã tồn tại giá trị hay chưa . nếu đã tồn tại giá trị thì trả về chính nó . còn chưa tồn tại hoặc có giá trị là nil thì sẽ được gán giá trị trả về ở biểu thức bên phải